src/metalearner.jl

```diff
@@ -71,8 +89,3 @@ function make_learner(args...; kw...)
     end
     MetaLearner(args..., strats...)
 end
-
-# add to an existing meta
-function make_learner(meta::MetaLearner, args...; kw...)
```

Comment:
Removing this looks like a breaking change. is it? Why did you remove this method?